codechain/config/presets/config.dev.toml

```diff
@@ -5,7 +5,7 @@ keys_path = "keys"
 chain = "solo"
 
 [mining]
-disable = false
+disable = true
```

Comment:
I think it's better to remove from the presets also.

Reply:
Do you mean to remove the `disable` preset default and set it after checking cli arguments?

Reply:
Yes. I do.

Reply:
Ok, I understand.